docs/usage/prometheus-metrics.md

```diff
@@ -27,3 +27,16 @@ We provide a {download}`sample Grafana dashboard </_downloads/grafana/liqonetwor
 As presented in the screenshot below, it includes an overview section presenting the overall cross-cluster throughput, followed by detailed per-peering throughput and latency information.
 
 ![Grafana Network Dashboard](/_static/images/usage/prometheus-metrics/network-dashboard.png)
+
+## Virtual kubelet metrics
+
+These metrics are available for each peered remote cluster, providing statistics about the reflected resources:
+
+- **liqo_virtual_kubelet_reflection_item_counter**: the number of reflected resources successfully processed.
+- **liqo_virtual_kubelet_reflection_error_counter**: the number of transient errors during the reflection phase.
```

Comment:
I would suggest to make an example here as well.
E.g.:
.... during the reflection phase, e.g., due to ....
(and list 2-3 reasons we may have errors here)

Reply:
Ok, I will add some examples.

Reply:
While discussing it with @aleoli, the main motivation behind the errors is that the issue is caused by temporary race conditions that can be resolved by retrying the synchronization. These conditions mainly occur when some of the requested resources are not yet fully configured. There could be three specific cases to discuss (e.g., reflector not completely initialized or fallback not completely initialized), but we think it would be too detailed to add to the documentation.

Reply:
@aleoli @damianot98 Reporting an error is something that *should* be interesting for the user. For me, as a customer, knowing that are errors and (a) knowing nothing about why they are here, (b) having no way to solve them, means that this counter is completely useless. Even worse, it makes me crazy, as I know that something is not working, and I cannot do anything to solve the problem.
Please consider this and improve the documentation according to the above points.

Reply:
@frisso I extended the previous example, providing the three main reasons why errors occur in the reflection stage. Although it is highly specific, it would be more helpful from a user point of view.